Softeq.XToolkit.Common.Tests/Extensions/InternalSettingsExtensionsTests/InternalSettingsExtensionsTests.cs

```diff
@@ -0,0 +1,99 @@
+﻿// Developed by Softeq Development Corporation
+// http://www.softeq.com
+
+using NSubstitute;
+using Softeq.XToolkit.Common.Extensions;
+using Softeq.XToolkit.Common.Interfaces;
+using Xunit;
+
+namespace Softeq.XToolkit.Common.Tests.Extensions.InternalSettingsExtensionsTests
+{
+    public class InternalSettingsExtensionsTests
+    {
+        [Fact]
+        public void AddOrUpdateJsonValue_NullValue_RemovesKey()
+        {
+            var internalSettings = Substitute.For<IInternalSettings>();
+            var jsonSerializer = Substitute.For<IJsonSerializer>();
+            var key = InternalSettingsExtensionsHelper.Key;
```

Comment:
Please use "Constructor and Dispose" for "Shared Context between Tests":
https://xunit.net/docs/shared-context